js/script.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-unused-vars */
+const titleInput = document.getElementById('title');
+const authorInput = document.getElementById('author');
+const addButton = document.getElementById('submit');
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Booklist {
+  constructor() {
+    this.books = [];
+    if (localStorage.books) this.books = JSON.parse(localStorage.getItem('books'));
+  }
+
+  refreshBookList() {
+    let listItems = '';
+    const booksSection = document.getElementById('books-section');
+    this.books.forEach((element, i) => {
+      const book = `<li class="single-book">
+                                 <p>"${this.books[i].title}" by ${this.books[i].author}</p>
+                                 <button type="button" class="remove" data-index = "${i}" onclick="removeButton(event)">Remove</button>
+                    </li>`;
+      listItems += book;
+    });
+
+    booksSection.innerHTML = listItems;
+  }
+}
+
+const booklist = new Booklist();
+booklist.refreshBookList();
+
+function addBook() {
+  const book = new Book(titleInput.value, authorInput.value);
+  booklist.books.push(book);
+  booklist.refreshBookList();
+  localStorage.setItem('books', JSON.stringify(booklist.books));
+  titleInput.value = '';
+  authorInput.value = '';
+}
+
+function removeButton(event) {
+  const indexBook = event.currentTarget.dataset.index;
+  booklist.books.splice(parseInt(indexBook, 10), 1);
+  booklist.refreshBookList();
+  localStorage.setItem('books', JSON.stringify(booklist.books));
+}
```

Comment:
- [ ] It would be nice if these methods are created inside of the class and then you can make use of them when you create an instance of the class. For this project, you're required to have class methods for adding and removing a todo. Please work on that. 👍